main/src/ca/uwaterloo/flix/api/lsp/provider/completion/Completion.scala

```diff
@@ -272,13 +272,13 @@ object Completion {
     * Represents a type completion for enum
     *
     * @param enumSym       the enum symbol.
-    * @param nameSuffix    the suffix for the name of the EnumType.
-    * @param priority      the priority of the EnumType.
+    * @param nameSuffix    the suffix for the name of the Enum.
+    * @param priority      the priority of the Enum.
     * @param textEdit      the edit which is applied to a document when selecting this completion.
     * @param documentation a human-readable string that represents a doc-comment.
     */
-  case class TypeEnumCompletion(enumSym: EnumSym, nameSuffix: String, priority: String, textEdit: TextEdit,
-                                documentation: Option[String]) extends Completion
+  case class EnumCompletion(enumSym: EnumSym, nameSuffix: String, priority: String, textEdit: TextEdit,
+                            documentation: Option[String]) extends Completion
```

Comment:
This could be cleaned up, however, I think that should be part of another PR, and IF I have time to do it.

Reply:
Yes, I agree :)